algrtm/src/components/CustomSolutionsSection.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import CustomLink from "./shared/CustomLink";
+
+const CustomSolutionsSection = () => {
+  return (
+    <section className="flex mt-[3.945rem] w-full h-[25.5625rem] relative">
+      <div className="flex flex-col gap-4 py-[4.66rem] pl-[6.62rem] pr-[0rem">
```

Comment:
Here the padding makes the area much larger than in Figma. I think we only need 200px on the left side, assuming we have removed the padding from the `<main>` element